src/index.html

```diff
@@ -11,7 +11,158 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <main>
+
+      <form
```

Comment:
Don't add empty line between parent and child tags
```suggestion
    <main>
      <form
```